app/src/main/java/com/conupods/OutdoorMaps/View/MainActivity.java

```diff
@@ -0,0 +1,74 @@
+package com.conupods.OutdoorMaps.View;
+
+import android.app.Dialog;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.conupods.R;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class MainActivity extends AppCompatActivity {
```

Comment:
Should the main activity be Part of the outdoors package? I dont think so

Reply:
It is not truely a "Main activity" it serves its purpose as a placeholder until sign in and sign ups have been done. Until then, we need a dummy page